meinberlin/apps/budgeting/templates/meinberlin_budgeting/includes/proposal_form.html

```diff
@@ -35,14 +35,20 @@ <h2>{% blocktrans %}Contact Information{% endblocktrans %}</h2>
 
     <div class="form-check">
         <label class="form-check__label">
-            {{ form.contact_storage_consent }}
+            {{ form.contact_storage_consent|add_error_attr:"aria-invalid:true" }}
             {% blocktrans with privacy_policy="/datenschutz" %}I hereby expressly consent to the storage of the contact information I have entered. The contact information may be used by the responsible administration exclusively for contacting me in the present project. I can revoke this consent at any time. I have read and accept the <a href="{{privacy_policy}}" target="_blank">privacy policy</a>.{% endblocktrans %}
         </label>
-        {{ form.contact_storage_consent.errors }}
+        {% if form.contact_storage_consent.errors %}
+            <ul class="errorlist">
+            {% for error in form.contact_storage_consent.errors %}
+                <li aria-live="assertive" aria-atomic="true" id="error-{{ form.contact_storage_consent.id_for_label }}-{{ forloop.counter }}">
```

Comment:
@phillimorland I have added it to the list elements, but have seen that we added it to the errorlist elsewhere. What's the right place to put it?

Reply:
@fuzzylogic2000 on the error list as the containing element, will also tell screen reader number of items on list instead of just list item